tsconfig.build.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file is for the v1 tokens. I've renamed this to tsconfig.build.json as it was causing compilation issues for everything else that uses TS.

Reply:
👍